by_id/odb.manualmacroplacement/003-invalid_location/handler.py

```diff
@@ -0,0 +1,15 @@
+import os
+import subprocess
+
+
+def handle(step, exception):
+    assert exception is not None, "Uninstantiated macro did not raise exception"
+    result = subprocess.call(
+        [
+            "grep",
+            "-E",
+            "Declared macros not instantiated in design",
```

Comment:
test folder is called `invalid_location`. How is that related to this message being grepped ?